include/circt/Target/Verilog/VerilogPrinter.h

```diff
@@ -0,0 +1,54 @@
+#ifndef CIRCT_TARGET_VERILOG_VERILOGPRINTER_H
+#define CIRCT_TARGET_VERILOG_VERILOGPRINTER_H
+
+#include "circt/Dialect/LLHD/IR/LLHDOps.h"
+
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/IR/Module.h"
+#include "llvm/ADT/Twine.h"
+#include "llvm/Support/FormattedStream.h"
+
+using namespace llvm;
+
+namespace mlir {
+namespace llhd {
+
+class VerilogPrinter {
```

Comment:
ditto - the verilog printer seems like it can be a couple of global functions as the public API in the header - these details can move into the lib directory.